infrastructure/core/main.tf

```diff
@@ -38,7 +38,7 @@ resource "azurerm_key_vault" "core" {
   resource_group_name           = azurerm_resource_group.core.name
   tenant_id                     = data.azurerm_client_config.current.tenant_id
   soft_delete_retention_days    = 7
-  purge_protection_enabled      = false
+  purge_protection_enabled      = var.tf_in_automation ? true : false # switch on purge protection for prod
```

Comment:
is this likely to become painful if infra-test needs to be destroyed and recreated? had that in MVP1

Reply:
yup - logged https://github.com/UCLH-Foundry/FlowEHR/issues/190